### DISCUSSION TITLE
Pushorigin issue

### DISCUSSION THREAD

**Discussion by bkandala2579** [community/community#22623] [New to GitHub]
<p>Hi All,</p>
<p>This is my first post in this forum. This is my first time using Github.  I am seeing an issue when i push my brnach. In github desktop i made changes and when i click on push origin i do not get an error message but after spinning the refresh button still shows push origin with a number next to it. When i got the git bash and use the command “git push” and come back to the desktop he push origin message is replaced with fetch origin. Please advice.</p>
<p>Thanks</p>

**Comment by mpboom** [comment]
<p>But is the branch actually pushed (if you open op <a href="http://github.com" rel="nofollow noopener">github.com</a>, do you see the changes)?</p>

**Comment by steveward** [comment, marked as answer]
<p><a class="mention" href="/bkandala2579">@bkandala2579</a> it sounds like the push is succeeding via the command line but not via GitHub Desktop. If you are continuining to encounter this issue I would be happy to debug it further to see if there are any underlying errors that are causing the push to fail. </p>
<p>Could you upload the log file from GitHub Desktop to see if that gives some more information about this issue?</p>
<p>To access the log files go to the file menu in GitHub Desktop and select <code>Help</code> &gt; <code>Show Logs in Finder (macOS) or Explorer (Windows)</code>.</p>
<p>The log files are created daily – please upload a log file as an attachment from a day where you experienced the issue. Thanks!</p>
